### DISCUSSION TITLE
winmerge declares folders are different, but they don't appear to me to be different

### DISCUSSION THREAD

**Discussion by whburling** [WinMerge/winmerge#694] [Q&A]
I am using w10 insider and Business 365 SharePoint.

I am comparing two SharePoints against each other. My expectations are that they should be the same.. 
The "left"    SharePoint is named:  Operations SharePoint - Documents. 
The "right" SharePoint is named:  All_RVC_Contributors - Documents

WinMerge identifies differences between many folders within the above two SharePoints.

for this discussion, I am only interested in two folders:
in the left SharePoint (Operations SharePoint - Documents) the folder is  
[AUTO INSURANCE CO. TELEPHONE NUMBERS.xlsx](https://github.com/WinMerge/winmerge/files/6199358/AUTO.INSURANCE.CO.TELEPHONE.NUMBERS.xlsx)

In the right SharePoint (All_RVC_Contributors - Documents) the folder is
[AUTO INSURANCE CO. TELEPHONE NUMBERS.xlsx](https://github.com/WinMerge/winmerge/files/6199365/AUTO.INSURANCE.CO.TELEPHONE.NUMBERS.xlsx)

Clearly WinMerge sees differences.  I can't see them when I view the files.  Can someone help me understand
what is happening?   

Are the differences due to unseen formatting?  How do I tell Winmerge to ignore formatting, if that is the case. 
As  you can see the actual text is the same. 

I have to understand this issue as I am comparing many many SharePoints and I can't open each file in each SharePoint
to compare them. Just too many.  

Thank you in advance.
bil

**Comment by whburling** [comment]
this is bil (author),

It appears that the differences have something to do with xsd information as the differences in the files of a folder reference xsd  at the end of the two files.  So my initial question comes down to how to tell winmerge to not pay attention to formatting but only to content. In my case, all content is (1) text in either word documents or excel documents or (2) images of text in word documents.

**Comment by sdottaka** [comment]
There is a difference in the footer of Sheet1.

![image](https://user-images.githubusercontent.com/98126/112397453-74fe1300-8d45-11eb-90dd-2ae83fd08af9.png)

WinMerge's CompareMSExcelFiles.sct plugin can extract and compare the text from Excel workbooks. This plugin can be set not to compare the header and footer.

To enable this plugin, click Plugins → Auto Unpacking menu item.

![image](https://user-images.githubusercontent.com/98126/112397824-2b61f800-8d46-11eb-8ef9-1aa10fc97da8.png)

To set not to compare the header and footer, click [Plugins] → [Plugin Settings] menu item and set as follows.

![image](https://user-images.githubusercontent.com/98126/112397846-33219c80-8d46-11eb-913e-703eecbe4d4e.png)

**Comment by whburling** [comment]
How do i ever thank you?

How can i help you in some way?

You reached out and helped me with no demand for compensation.  I live that way but not experienced life that way from others.

Thank you for your taking your time to help me understand the problem snd then show me how to resolve it. Wow.

bil

Sent by cell phone


On Mar 24, 2021, at 19:49, Takashi Sawanaka ***@***.***> wrote:

﻿

There is a difference in the footer of Sheet1.

[image]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F98126%2F112397453-74fe1300-8d45-11eb-90dd-2ae83fd08af9.png&data=04%7C01%7C%7Cbad47c5e697848139a2e08d8ef1f7a5c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637522265757371835%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=%2F52mL1rOJ5BA79COlPnG3MAU%2FvpayoHWblvTH%2BU02ng%3D&reserved=0>

WinMerge's CompareMSExcelFiles.sct plugin can extract and compare the text from Excel workbooks. This plugin can be set not to compare the header and footer.

To enable this plugin, click Plugins → Auto Unpacking menu item.

[image]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F98126%2F112397824-2b61f800-8d46-11eb-8ef9-1aa10fc97da8.png&data=04%7C01%7C%7Cbad47c5e697848139a2e08d8ef1f7a5c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637522265757371835%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=oakMT6DUEDkCBDhM%2Bsb9AV0R33OO1mnyS64ZTUB1YNk%3D&reserved=0>

To set not to compare the header and footer, click [Plugins] → [Plugin Settings] menu item and set as follows.

[image]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F98126%2F112397846-33219c80-8d46-11eb-913e-703eecbe4d4e.png&data=04%7C01%7C%7Cbad47c5e697848139a2e08d8ef1f7a5c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637522265757381833%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=rZ9nz8LdkHhD%2FDAz5FlE%2Fo19zuG%2BmJksMH6Sc2G%2Ba3E%3D&reserved=0>

**Comment by whburling** [comment]
How do i see the excel spread sheets as you did
Which exposed the footer info?

Sent by cell phone


On Mar 24, 2021, at 19:49, Takashi Sawanaka ***@***.***> wrote:

﻿

There is a difference in the footer of Sheet1.

[image]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F98126%2F112397453-74fe1300-8d45-11eb-90dd-2ae83fd08af9.png&data=04%7C01%7C%7Cbad47c5e697848139a2e08d8ef1f7a5c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637522265757371835%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=%2F52mL1rOJ5BA79COlPnG3MAU%2FvpayoHWblvTH%2BU02ng%3D&reserved=0>

WinMerge's CompareMSExcelFiles.sct plugin can extract and compare the text from Excel workbooks. This plugin can be set not to compare the header and footer.

To enable this plugin, click Plugins → Auto Unpacking menu item.

[image]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F98126%2F112397824-2b61f800-8d46-11eb-8ef9-1aa10fc97da8.png&data=04%7C01%7C%7Cbad47c5e697848139a2e08d8ef1f7a5c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637522265757371835%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=oakMT6DUEDkCBDhM%2Bsb9AV0R33OO1mnyS64ZTUB1YNk%3D&reserved=0>

To set not to compare the header and footer, click [Plugins] → [Plugin Settings] menu item and set as follows.

[image]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F98126%2F112397846-33219c80-8d46-11eb-913e-703eecbe4d4e.png&data=04%7C01%7C%7Cbad47c5e697848139a2e08d8ef1f7a5c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637522265757381833%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=rZ9nz8LdkHhD%2FDAz5FlE%2Fo19zuG%2BmJksMH6Sc2G%2Ba3E%3D&reserved=0>

**Reply by sdottaka** [reply]
Please re-compare the Excel book file with the settings described at https://github.com/WinMerge/winmerge/discussions/694#discussioncomment-527351.
